docs/zh-cn/development/pull-request.md

```diff
@@ -0,0 +1,53 @@
+## 前言
+Pull Request 本质上是一种软件的合作方式，是将涉及不同功能的代码，纳入主干的一种流程。这个过程中，可以进行讨论、审核和修改代码。
+
+在 Pull Request 中尽量不讨论代码的实现方案，代码及其逻辑的大体实现方案应该尽量在
+Issue 中被确定，在 Pull Request 中我们尽量只关注代码的格式以及代码规范等信息，从而避免实现方式的意见不同而导致
+waste time。
+
+## 规范
+
+### Pull Request 标题
+
+标题格式：[`Pull Request 类型`-`Issue 号`][`模块名`] `Pull Request 描述`
+
+其中`Pull Request 类型`和`Issue 类型`的对应关系如下：
+
+<table>
+    <thead>
+        <tr>
+            <th style="width: 10%; text-align: center;">Issue 类型</th>
+            <th style="width: 20%; text-align: center;">Pull Request 类型</th>
+            <th style="width: 20%; text-align: center;">样例（假设 Issue 号为 3333）</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="text-align: center;">Feature</td>
+            <td style="text-align: center;">Feature</td>
+            <td style="text-align: center;">[Feature-3333][server] Implement xxx</td>
+        </tr>
+        <tr>
+            <td style="text-align: center;">Bug</td>
+            <td style="text-align: center;">Hotfix</td>
+            <td style="text-align: center;">[Hotfix-3333][server] Fix xxx</td>
```

Comment:
I think "fix" is much better than Hotfix, there is not all bug is belong to Hotfix? how do you think ?

Reply:
I agree with you. I will change it.

Reply:
> I think "fix" is much better than Hotfix, there is not all bug is belong to Hotfix? how do you think ?

I agree with you. I will change it.